lib/screens/container_view.dart

```diff
@@ -9,15 +12,117 @@ class ContainerView extends StatefulWidget {
   _ContainerViewState createState() => _ContainerViewState();
 }
 
+enum MyBorderRadius {
+  round,
+  left,
+  right,
+  top,
+  bottom,
+  topLeft,
+  topRight,
+  bottomLeft,
+  bottomRight,
+  circle,
+}
+
+enum MyGradient {
+  linearGradient,
+  radialGradient,
+  sweepGradient,
+}
+
 class _ContainerViewState extends State<ContainerView> {
   bool _showBorder = true;
+  bool _showShadow = true;
+  // Map<String, BorderRadius> listBorder = {
+  //   'round': BorderRadius.circular(20),
+  // };
+  List<BorderRadius> listRadius = [
+    BorderRadius.circular(20),
+    const BorderRadius.horizontal(left: Radius.circular(20)),
+    const BorderRadius.horizontal(right: Radius.circular(20)),
+    const BorderRadius.vertical(top: Radius.circular(20)),
+    const BorderRadius.vertical(bottom: Radius.circular(20)),
+    const BorderRadius.only(topLeft: Radius.circular(20)),
+    const BorderRadius.only(topRight: Radius.circular(20)),
+    const BorderRadius.only(bottomLeft: Radius.circular(20)),
+    const BorderRadius.only(bottomRight: Radius.circular(20)),
+    BorderRadius.circular(200),
+  ];
```

Comment:
Change to Map<String, BorderRadius>
Ex: Map<String, Bor..> mapRadius = {
         "round": BorderRadius.circular(20),
         "left": const BorderRadius.horizontal(left: Radius.circular(20)), ...}